### PR TITLE
simplify OBU_length_mode (#3)

### PR DESCRIPTION
**Pull request by cconcolato**
Removes issues related to the OBU_length_mode from the spec to address issue #3

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -145,8 +145,8 @@ class AV1CodecConfigurationBox extends FullBox('av1C', version = 0, 0){
 }
 
 aligned (8) class AV1CodecConfigurationRecord {
-  unsigned int (6) reserved = 0;
-  unsigned int (2) OBU_length_mode;
+  unsigned int (7) reserved = 0;
```
Comment (tomfinegan) [inline index.bs]: 7 + 1 + 3 = 11?

The 7 should be 4, no? Missed this in the last pass.

**Comment by tdaede** [comment]
LGTM. I'd like to drop it entirely of course but this is still better.

File: index.bs (hunk 1 above)
Comment (cconcolato) [reply index.bs]: indeed ...

**Comment by cconcolato** [comment]
For the record, this issue was discussed during the AV1 RTC call today, it was mentioned that one concern regarding OBU_length_mode 1/2 was for RTP recording into ISOBMFF, but that there was no objection if we felt that having modes was a problem. Also, it was mentioned that the ISOBMFF base spec already defines "RTP Reception Hint Track" for recording. In the end, it was decided that unless objections in the coming days, the OBU_length_mode would be dropped, only mode 1/2 would be used.

**Comment by cconcolato** [comment]
Replaced by #15

**Pull request closed without merging**